cmake/modules/FindUFE.cmake

```diff
@@ -46,6 +46,10 @@ if(UFE_INCLUDE_DIR AND EXISTS "${UFE_INCLUDE_DIR}/ufe/ufe.h")
 
     if("${UFE_MAJOR_VERSION}" STREQUAL "0")
         math(EXPR UFE_PREVIEW_VERSION_NUM "${UFE_MINOR_VERSION} * 1000 + ${UFE_PATCH_LEVEL}")
+    elseif("${UFE_VERSION}" STREQUAL "2.0.0")
+        # Temporary. Once next Maya PR is released with UFE v2.0.0 this should
+        # be removed (along with all the UFE_PREVIEW_VERSION_NUM checks).
+        set(UFE_PREVIEW_VERSION_NUM 2036)
```

Comment:
Internally switching to UFE v2.0.0 means that line 47 is not true anymore so UFE_PREVIEW_VERSION_NUM will no longer be set. This is a temporary measure to have it set to the last ufe version (0.2.36).